cylc/flow/network/scan.py

```diff
@@ -271,10 +273,24 @@ def get_scan_items_from_fs(
 
             # Choose only suites with .service and matching filter
             if active_only:
+                # Skip suites running with cylc version < 8 (these suites
+                # do not have PUBLISH_PORT field)
                 try:
-                    contact_data = load_contact_file(
-                        reg, owner)
-                except (SuiteServiceFileError, IOError, TypeError, ValueError):
+                    contact_data = load_contact_file(reg, owner)
+                except (SuiteServiceFileError, IOError, TypeError) as exc:
+                    LOG.debug(f"Error loading contact file for: {reg}")
+                    continue
+                try:
+                    cylc_version = contact_data[ContactFileFields.VERSION]
+                    major_version = int(cylc_version.split(".", 1)[0])
+                    if (major_version < 8):
+                        LOG.info(f"Suite \"{reg}\" is running in Cylc version"
+                                 f" {cylc_version}, not Cylc 8 and"
+                                 f" will not be displayed.")
```

Comment:
IMO that's too verbose to print to the terminal. How about just this?:
```suggestion
                        LOG.info(f"Omitting \"{reg}\" (cylc-{cylc_version})")
```